.golangci.yaml

```diff
@@ -60,9 +55,7 @@ linters:
     - noctx
     - nolintlint
     - staticcheck
-    - structcheck
     - stylecheck # replacement for golint
     - unconvert
     - unused
-    - varcheck
```

Comment:
These linters are deprecated, with exception of `gofmt`. 

`gofmt` is duplicated via `goimports`, so removing it.